docker/local-development.md

```diff
@@ -299,6 +203,22 @@ docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app <Y
 
 – where `<YOUR_COMMAND>` is something like `pytest tests/test_admin.py -sxv --pdb`.
 
+Commands that write to STDOUT, e.g., `python manage.py dumpdata`, can be piped (`|`)
+or redirected (`>`) to files on your computer in the normal way.
+
+If you need to run a command that writes to files, e.g., `python manage.py makemigrations`,
+and you'd like those files to be written to your host machine, you can use this trick:
+
+```bash
+docker-compose run --no-deps --rm -v `pwd`:`pwd` -w `pwd` app python manage.py makemigrations -n mymigration
```

Comment:
Two notes:

1. I don't think `-v pwd:pwd -w pwd` is necessary in this case because we typically mount the application directory into the container and define the working directory in `docker-compose.yml`. If I understand right, these flags are necessary for the cookiecutter command because we're running a bare `docker` command in that case.
2. `--no-deps` seems wrong to me here because we need the database service in order to make migrations.

I think the adjusted command should be:

```suggestion
docker-compose run --rm app python manage.py makemigrations -n mymigration
```

However, since the purpose of this section is to try to explain how to write files to the container, which is unnecessary if 1) above is true, then I think we can just go ahead and scrap L209-L220 entirely instead of fixing this command.